WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/support/WPScreenshotIdler.java

```diff
@@ -0,0 +1,56 @@
+package org.wordpress.android.ui.screenshots.support;
+
+import android.support.test.espresso.IdlingResource;
+import static junit.framework.Assert.fail;
+
+public abstract class WPScreenshotIdler implements IdlingResource {
```

Comment:
This class still conforms to `IdlingResource` because if it's ever possible to use Espresso assertions within an `IdlingResource`, it'd be trivial to register it and eliminate the need for `idleUntilReady()`, but that feels a little like over-engineering things for a future that might never exist.

I'm on the fence about it!